tested/testsuite.py

```diff
@@ -528,7 +535,7 @@ def get_functions(self) -> Iterable[FunctionCall]:
 
 @define(frozen=True)
 class FileUrl:
-    url: str
+    path: str
```

Comment:
The reason this was named `url` is that this is a path to the media folder of an exercise, since it is linked on Dodona. The "generated "file" stuff should all have paths to the `evaluation` folder instead. Now, there is an argument to be made that this distinction is unnecessary, but that is done at the Dodona side (I think, it has been a while, so this should be checked).

But to be clear, I am not against renaming this, just giving some context.